test/t_cose_test.c

```diff
@@ -2230,3 +2232,95 @@ int_fast32_t crypto_context_test()
 
     return 0;
 }
+
```

Comment:
This .c file is for tests that only need the test crypto adapter. There are no tests in it that need PSA or OSSL, so this test shouldn't be here, but in t_cose_sign_verify_test.c instead.

Reply:
Done